CONTRIBUTING.md

```diff
@@ -97,7 +97,7 @@ If the pull request closes an issue please note it as: `"Fixes #NNN"`.
 ## Code Reviews
 
 Look at the area of code you're modifying, its history, and consider
-tagging some of the [maintainers](MAINTAINERS.md) when doing a
+tagging some of the [maintainers](http://u-root.tk/#contributors) when doing a
```

Comment:
It would be even better if you make this the https link.

Reply:
Updated link to https.
Not sure which workflow you prefer in your project. For this time that was 'commit --amend'